examples/mq/v1alpha1/user.yaml

```diff
@@ -0,0 +1,88 @@
+# SPDX-FileCopyrightText: 2024 The Crossplane Authors <https://crossplane.io>
+#
+# SPDX-License-Identifier: CC0-1.0
+
+apiVersion: mq.aws.upbound.io/v1alpha1
+kind: User
+metadata:
+  annotations:
+    meta.upbound.io/example-id: mq/v1alpha1/user
+  labels:
+    testing.upbound.io/example-name: example
+  name: example
+spec:
+  forProvider:
+    brokerIdSelector:
+      matchLabels:
+        testing.upbound.io/example-name: example-user
+    # setting consoleAccess will result in an update loop till the broker is
+    # restarted...
+    # consoleAccess: true
+    passwordSecretRef:
+      key: password
+      name: mq-secret
+      namespace: upbound-system
+    username: test-user
+    region: us-west-1
+
+---
+
+apiVersion: mq.aws.upbound.io/v1beta1
+kind: Broker
+metadata:
+  annotations:
+    meta.upbound.io/example-id: mq/v1alpha1/user
+  labels:
+    testing.upbound.io/example-name: example-user
+  name: user-example-broker
+spec:
+  forProvider:
+    # Due to an existing Terrajet limitation, we cannot use "metadata.name" for "brokerName"
+    # Details can be found in https://github.com/crossplane/terrajet/issues/280
+    brokerName: user-example-broker
+    region: us-west-1
+    securityGroupRefs:
+      - name: example
+    engineType: ActiveMQ
+    engineVersion: 5.17.6
+    hostInstanceType: mq.t3.micro
+    user:
```

Comment:
Let's move this under to `spec.initProvider`. Please also see: https://github.com/crossplane/upjet/pull/406

Reply:
Let's postpone this change until after referenced Upjet PR is merged. Proposed change would render the example manifest unusable, because `spec.initProvider` doesn't have `user` field at the moment.